Dockerfile

```diff
@@ -35,11 +35,12 @@ COPY contrib/* ./share/valheim/contrib/
 FROM debian:stable-slim
 ENV DEBIAN_FRONTEND=noninteractive
 COPY --from=build-env /artifacts/ /usr/local/
+COPY supervisord.conf /etc/supervisor/supervisord.conf.valheim
```

Comment:
Instead of copying this file on its own, we should copy it during the first build stage into `/artifacts`, and then `mv` and `chmod` it in the `RUN` command below during this second stage to avoid creating an extra unnecessary layer.  The file itself may be small, but Docker image layers aren't free and we should try to minimize the number of layers in the final image 🙂  Not really a huge deal, but just best practice.

Reply:
Great idea! I also want to optimize the redundant COPYs in the build-env but need to fix the paths first. But I'm going to sleep now. 5:22 am here. Tomorrow! 🙂

Reply:
@lloesche The `build-env` isn't as important since it basically gets thrown away, but yeah it'd be nice to just `COPY` once there!

Reply:
Yup I didn't mean from a Docker point of view but from a maintainability one. It's error prone to have the scripts in there twice. Whenever we add or remove one we have to remember to do it in both places.